pages/dkp/kommander/2.2/backup-and-restore/index.md

```diff
@@ -197,6 +197,46 @@ To restore cluster data on demand from a selected backup snapshot available in t
 velero restore create --from-backup BACKUP-NAME
 ```
 
+#### Restoring Kommander Management Cluster
+
+<p class="message--note"><strong>NOTE: </strong>When restoring a backup to the Kommander Management Cluster, you must adjust configuration to avoid restore errors.
```

Comment:
```suggestion
<p class="message--note"><strong>NOTE: </strong>When restoring a backup to the DKP Management Cluster, you must adjust configuration to avoid restore errors.
```

Reply:
The rest of the document still uses the term `Kommander` so I think we should keep it and update with the rest of the documentation.

Reply:
Agreed, we should review entire documents and not single lines when changing to DKP wording.